apps/src/templates/certificates/CertificateBatch.jsx

```diff
@@ -2,9 +2,73 @@ import React, {useState} from 'react';
 import PropTypes from 'prop-types';
 import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
 import i18n from '@cdo/locale';
-import styleConstants from '../../styleConstants';
-import color from '@cdo/apps/util/color';
 import RailsAuthenticityToken from '@cdo/apps/lib/util/RailsAuthenticityToken';
+import style from './certificate_batch.module.scss';
+
+const curriculaData = [
+  {
+    grade: i18n.gradeRange({
+      numGrades: ['3', '8'].length,
```

Comment:
I think you may have meant `[3...8].length` since `['3', '8'].length` will return 2 (as will all the subsequent `numGrades`). But since you are getting the length of a hard-coded array, I think it may make more sense to juts hard-code `numGrades` to however many grades there are.